app/core/database.py

```
@@ -12,5 +12,8 @@
 
 # 의존성 주입
 async def get_db():
-    async with async_session() as session:
-        yield session
+    try:
+        async with async_session() as session:
+            yield session
+    except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)